caluma/caluma_core/tests/test_set_path.py

```diff
@@ -0,0 +1,21 @@
+def test_set_paths(db, case, work_item_factory):
```

Comment:
Please put this test into the `caluma_workflow` app, because it depends on it as well as on the `caluma_form` app.

Reply:
Done, ready for re-review